### PR TITLE
Feature/16477 tde mapping

### PR DESCRIPTION
**Pull request by stachow**

### CODE REVIEW

File: rumpole-gateway/Domain/RumpolePipeline/DocumentStatus.cs [hunk 1]
```diff
@@ -5,10 +5,12 @@ public enum DocumentStatus
 		None,
 		PdfUploadedToBlob,
 		Indexed,
-		NotFoundInCDE,
```
Comment (stachow) [inline rumpole-gateway/Domain/RumpolePipeline/DocumentStatus.cs:8]: @aceltuk is this correct that these enum flags are coming in with this merge, or am I bringing back dead code?

File: rumpole-gateway/rumpole-gateway.csproj [hunk 2]
```diff
@@ -22,7 +24,7 @@
     <PackageReference Include="Microsoft.Identity.Client" Version="4.46.2" />
     <PackageReference Include="Microsoft.IdentityModel.Protocols" Version="6.23.0" />
     <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="6.23.0" />
-    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.1.3" />
```
Comment (stachow) [inline rumpole-gateway/rumpole-gateway.csproj:25]: @aceltuk I couldn't get the gateway to compile without reverting to an older version of this package.  Is this ok to commit for the time being or is it daft?

**Review by aceltuk — APPROVED** [review]
All looks good to me

File: rumpole-gateway/rumpole-gateway.csproj (hunk 2 above)
Comment (aceltuk) [reply rumpole-gateway/rumpole-gateway.csproj:25]: I think this will be OK

**Pull request merged**